checks/apps/cp2k/cp2k_uenv.py

```diff
@@ -295,6 +316,7 @@ class Cp2kCheckRPA_UENVExec(Cp2kCheck_UENV):
     executable_opts = ['-i', 'H2O-128-RI-dRPA-TZ.inp']
     energy_reference = -2217.36884935325
     tags = {'maintenance'}
+    valid_systems = ['daint']
```

Comment:
line 164 should be:
```
valid_systems = ['+uenv']
```

Reply:
This test should only run on GH200 systems. What is the best way to express that condition?